sync-version/action.yaml

```
@@ -12,6 +12,9 @@ inputs:
   token:
     required: true
     description: A token with write privileges to the repository.
+  snapcraft-yaml-path:
+    description: "Custom path to snapcraft.yaml allowing to have multiple manifests in one repo."
```

I think this makes more sense as an explanation

```suggestion
    description: "Custom path to snapcraft.yaml for when it is not in the default location."
```